src/test/java/uk/gov/companieshouse/insolvency/delta/mapper/CaseMapperTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.companieshouse.insolvency.delta.mapper;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static uk.gov.companieshouse.api.delta.CaseNumber.CaseTypeEnum.ADMINISTRATION;
+import static uk.gov.companieshouse.api.delta.CaseNumber.CaseTypeEnum.RECEIVER_MANAGER;
+import static uk.gov.companieshouse.api.insolvency.CaseDates.TypeEnum.ADMINISTRATION_STARTED_ON;
+import static uk.gov.companieshouse.api.insolvency.ModelCase.TypeEnum.ADMINISTRATION_ORDER;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.companieshouse.api.delta.CaseNumber;
+import uk.gov.companieshouse.api.insolvency.CaseDates;
+import uk.gov.companieshouse.api.insolvency.ModelCase;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {
+        CaseMapperImpl.class,
```

Comment:
Not sure if we need SpringExtension and autowiring to test this? if so lets keep it but ideally each unit should be testable on its own if possible

Reply:
It is necessary as per the mapstruct documentation, we obtain the mapper via dependency injection so the unit tests need this - verified as they all failed when we didn't have this.